### PR TITLE
TW 236: add UI config variables

### PR DESCRIPTION
**Pull request by DJSanti**
This PR adds four new configuration variables to the Web UI config reference, along with additional information for `disableWriteActions`.

### CODE REVIEW

File: docs/references/web-ui-configuration.md [hunk 1]
```diff
@@ -160,6 +160,29 @@ codec:
...
 ```
 
-# disableWriteActions
+## disableWriteActions
 
-Prevents the user from terminating or canceling Workflow Executions from the Web UI.
\ No newline at end of file
+Prevents the user from executing Workflow Actions on the Web UI.
+
+This option affects Bulk Actions for Recent Workflows as well as Workflow Actions.
```
Comment (rossedfort) [inline docs/references/web-ui-configuration.md:167]: I'd consider adding "on the Workflow Details page" at the end of this sentence.

File: docs/references/web-ui-configuration.md [hunk 2]
```diff
@@ -160,6 +160,29 @@ codec:
...
+
+:::note 
+`disableWriteActions` overrides the configuration values of each individual Workflow Action. 
+Setting this variable to `true` will disable all options on the Web UI.
```
Comment (rossedfort) [inline docs/references/web-ui-configuration.md:171]: I'd replace `options` with `actions`

**Comment by DJSanti** [comment]
Okay, so my changes just disappeared. Hang on a moment...

**Comment by DJSanti** [comment]
@rossedfort changes should be back now. Going to go add the little yaml snippets and it should be good to go.

File: docs-src/references/web-ui-configuration.md [hunk 3]
```diff
@@ -158,6 +158,49 @@ codec:
...
+
+Prevents the user from canceling Workflow Executions from the Web UI.
+
+```yaml
+workflowCancelDisabled: false
+```
+
+## workflowSignalDisabled
+
+When enabled, prevents the Workflow from sending or receiving Signals from the Web UI.
```
Comment (rossedfort) [inline docs-src/references/web-ui-configuration.md:194]: I think we can just change this to: `Prevents the user from Signaling workflows from the Web UI.` to keep it more similar to the others.

File: docs/references/web-ui-configuration.md [hunk 4]
```diff
@@ -160,6 +160,49 @@ codec:
...
+
+Prevents the user from canceling Workflow Executions from the Web UI.
+
+```yaml
+workflowCancelDisabled: false
+```
+
+## workflowSignalDisabled
+
+When enabled, prevents the Workflow from sending or receiving Signals from the Web UI.
```
Comment (rossedfort) [inline docs/references/web-ui-configuration.md:196]: Same here

Comment (DJSanti) [reply docs/references/web-ui-configuration.md:196]: this will get changed once I run yarn gen again.

**Review by rossedfort — APPROVED** [review]

File: docs-src/references/web-ui-configuration.md [hunk 5]
```diff
@@ -158,6 +158,49 @@ codec:
...
+
+This option affects Bulk Actions for Recent Workflows as well as Workflow Actions on the Workflow Details page.
+
+```yaml
+disableWriteActions: false
+```
+
+:::note
+`disableWriteActions` overrides the configuration values of each individual Workflow Action.
+Setting this variable to `true` will disable all Workflow Actions on the Web UI.
```
Comment (djmagee) [inline docs-src/references/web-ui-configuration.md:173]:
```suggestion
Setting this variable to `true` disables all Workflow Actions on the Web UI.
```

**Review by djmagee — APPROVED** [review]

**Pull request merged**